packages/suite/src/reducers/suite/modalReducer.ts

```diff
@@ -97,4 +99,16 @@ const modalReducer = (state: State = initialState, action: Action): State => {
     }
 };
 
+export const selectTxStakeName = (state: AppState) => {
```

Comment:
I think it does not belong to this file

Reply:
Could you help me to find a proper place for this function? The project is pretty large, so it's hard to see it yet.